src/rt/msvc.d

```diff
@@ -0,0 +1,231 @@
+/**
+ * This module provides MS VC runtime helper functions that
+ * wrap differences between MS C runtime versions.
+ *
+ * Copyright: Copyright Digital Mars 2015.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Source:    $(DRUNTIMESRC rt/_msvc.d)
+ * Authors:   Rainer Schuetze
+ */
+module rt.msvc;
+
+version (CRuntime_Microsoft):
+
+import core.stdc.stdarg : va_list;
+import core.stdc.stdio : FILE, stdin, stdout, stderr, _vsnprintf;
+
+extern(C):
+nothrow:
+@nogc:
+
+struct _iobuf
+{
+    char* _ptr;
+    int   _cnt;  // _cnt and _base exchanged for VS2015
+    char* _base;
+    int   _flag;
+    int   _file;
+    int   _charbuf;
+    int   _bufsiz;
+    char* _tmpfname;
+    // additional members in VS2015
+}
+
+FILE* __acrt_iob_func(int hnd);     // VS2015+
+FILE* __iob_func();                 // VS2013-
+
+int _set_output_format(int format); // VS2013-
+
+immutable void* _nullfunc = null;
+
+__gshared ubyte msvcUsesUCRT;
+
+version (X86)
+    enum cPrefix = "_";
+else
+    enum cPrefix = "";
+
+mixin template declareAlternateName(string name, string alternateName)
+{
+    mixin(`pragma(linkerDirective, "/alternatename:` ~ cPrefix~name ~ `=` ~ cPrefix~alternateName ~ `");`);
```

Comment:
mixin accepts a comma separated list of strings

Reply:
I know, but it doesn't seem like an improvement to me:
```D
    mixin(`pragma(linkerDirective, "/alternatename:`, cPrefix, name, `=`, cPrefix, alternateName, `");`);